src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import HoneycombGrid from "./HoneycombGrid.style";
+
+function IntegrationsGrid({ obj }) {
+  return (
+    <HoneycombGrid>
+      <div className="container">
+        {obj.map((item) => {
+          const status = item.status === "InProgress" ? true : false;
+          if (status) {
+            return (
+              <div style={{ background: "rgba(160, 170, 170, 0.5)" }}>
+                <div style={{ background: "#F0F0F0" }}>
+                  <img
+                    src={item.img}
+                    alt={item.name}
+                    style={{ filter: "brightness(0) invert(1)" }}
+                  />
+                </div>
+              </div>
+            );
+          } else {
+            return (
+              <div>
+                <div>
+                  <img src={item.img} alt={item.name} />
+                </div>
+              </div>
```

Comment:
I don't think we need nested `div` here

Reply:
I think we do need it cause you can't make a border in clip-path div. So, to do that, I just made another div with a white background that acts as a border and a content holder.

Reply:
I believe, you can make use of pseudo classes for that.

Reply:
Let me check what I can do to fix this issue

Reply:
I am not able to fix that issue using pseudo-class. So if you can help me with that, it would be great.